lib/hMap.ml

```diff
@@ -247,6 +247,8 @@ struct
       let m = Map.singleton k x in
       Int.Map.add h m s
 
+  let update = assert false (* not implemented *)
```

Comment:
Don't do this, just don't expose update until someone needs it enough to implement it or we're on 4.06.
Also this will crash coq on startup since it's executed during initialisation.

Reply:
That's what I said in my comment -- this PR should not be merged until we're on 4.06-at-minimum.

Reply:
Oh, and I should add parameters.